src/window.ts

```diff
@@ -234,4 +250,139 @@ export class PersistedWindowLayout {
     constructor(name?: string) {
         this.name = name;
     }
-}
\ No newline at end of file
+}
+
+export class SnapAssistWindowManager {
+    private readonly container: Container;
+    private readonly floater: ContainerWindow;
+    public snapThreshold: number = 20;
+    public snapOffset: number = 15;
```

Comment:
This default of 15 works on Windows 10.  A different offset can be provided in the optional options to the constructor.

Reply:
https://github.com/Morgan-Stanley/desktopJS/issues/100